### PR TITLE
docs: update uninstallation guide

### PR DESCRIPTION
**Pull request by dkwon17**
Signed-off-by: David Kwon <dakwon@redhat.com>

This PR updates the uninstallation guide.

## What does this pull request change

Before:
image

After:
image

The PR also adds instructions to uninstall the DevWorkspace Operator resources before uninstalling Che.  Those instructions originally come from: https://docs.openshift.com/container-platform/4.8/web_console/odc-about-web-terminal.html#deleting-the-devworkspace-operator-dependency

## What issues does this pull request fix or reference
https://github.com/eclipse/che/issues/21284

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [x] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/uninstalling-che-after-operatorhub-installation.adoc [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+$ oc delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait
+----
+
++
+[subs="+quotes,+attributes,+macros"]
+----
+$ oc delete devworkspaceroutings.controller.devfile.io --all-namespaces --all --wait
+----
+
+pass:[<!-- vale RedHat.Spelling = NO -->]
```
Comment (dkwon17) [inline modules/administration-guide/pages/uninstalling-che-after-operatorhub-installation.adoc]: I have added this to ignore this warning:
```
 modules/administration-guide/pages/uninstalling-che-after-operatorhub-installation.adoc
 51:31  warning  Use correct American English    RedHat.Spelling 
                 spelling. Did you really mean                   
                 'finalizers'? 
```

**Review by ibuziuk — APPROVED** [review]

**Comment by max-cx** [comment]
I am reviewing this PR.

File: modules/administration-guide/pages/uninstalling-che.adoc [hunk 2]
```diff
@@ -12,10 +12,8 @@
 
 This section describes uninstallation procedures for {prod}. The uninstallation process leads to a complete removal of {prod-short}-related user data. The method previously used to install the {prod-short} instance determines the uninstallation method.
 
-* For {prod-short} installed using OperatorHub, for the OpenShift Web Console method see xref:uninstalling-che-after-operatorhub-installation-using-openshift-web-console.adoc[].
+* For {prod-short} installed using OperatorHub, see xref:uninstalling-che-after-operatorhub-installation.adoc[].
 
-* For {prod-short} installed using OperatorHub, for the CLI method see xref:uninstalling-che-after-operatorhub-installation-using-openshift-cli.adoc[].
-
-* For {prod-short} installed using {prod-cli}, see xref:uninstalling-che-after-chectl-installation.adoc[]
+* For {prod-short} installed using {prod-cli}, see xref:uninstalling-che-after-chectl-installation.adoc[].
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che.adoc]:
```suggestion
* For {prod-short} installed using `{prod-cli}`, see xref:uninstalling-che-after-chectl-installation.adoc[].
```

**Comment by max-cx** [comment]
I opened https://github.com/dkwon17/che-docs/pull/8 for my review.

File: modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+. Remove the {prod-short} instance from the {orch-namespace}:
+
++
+`$ {prod-cli} server:delete -n __<{orch-namespace}>__`
+
++
+[NOTE]
+====
+If uninstalling {prod-short} from a Minikube cluster, optionally remove the `dex` namespace.
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc]:
```suggestion
If uninstalling {prod-short} from a Minikube cluster: remove the `dex` and Che namespaces.
```

File: modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+`$ {orch-cli} delete namespaces dex`
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc]:
```suggestion
`$ {orch-cli} delete namespaces dex __<{orch-namespace}>__`
```

File: modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
++
+`$ {orch-cli} get checluster --all-namespaces -o=jsonpath="{.items[*].metadata.namespace}"`
+
+. Remove the {prod-short} instance from the {orch-namespace}:
+
++
+`$ {prod-cli} server:delete -n __<{orch-namespace}>__`
+
++
+include::example$snip_{project-context}-note-uninstalling-in-minicube-cluster.adoc[]
```
Comment (max-cx) [inline modules/administration-guide/pages/uninstalling-che-by-using-chectl.adoc]:
```suggestion
include::example$snip_{project-context}-note-uninstalling-in-minikube-cluster.adoc[]
```

**Pull request merged**